src/currencymanager.cpp

```diff
@@ -237,6 +237,9 @@ CurrencyDialog::CurrencyDialog(CurrencyManager& manager) : currency_manager_(man
     UpdateTotalWisdomValue();
     layout_->addWidget(new QLabel("Total Scrolls of Wisdom"), curr_row + 1, 0);
     layout_->addWidget(total_wisdom_value_, curr_row + 1, 1);
+    button_close_ = new QPushButton("Close");
```

Comment:
Yep, see http://imgur.com/29AwgrA
I could also override the windowFlags, but that could lead to unexpected behaviour on other OS/WM. This way it's non intrusive and works everywhere